pkg/apis/sources/v1alpha1/githubsource_types.go

```diff
@@ -88,14 +88,23 @@ type SecretValueFromSource struct {
 }
 
 const (
-	// gitHubSourceEventPrefix is what all GitHub event types get
+	// gitHubEventTypePrefix is what all GitHub event types get
 	// prefixed with when converting to CloudEvents.
-	gitHubSourceEventPrefix = "dev.knative.source.github"
+	gitHubEventTypePrefix = "dev.knative.source.github"
+
+	// gitHubEventSourcePrefix is what all GitHub event sources get
+	// prefixed with when converting to CloudEvents.
+	gitHubEventSourcePrefix = "//github.com"
 )
 
-// GetGitHubSourceEventType returns the GitHub CloudEvent type value.
-func GetGitHubSourceEventType(ghEventType string) string {
-	return fmt.Sprintf("%s.%s", gitHubSourceEventPrefix, ghEventType)
+// GitHubEventType returns the GitHub CloudEvent type value.
+func GitHubEventType(ghEventType string) string {
+	return fmt.Sprintf("%s.%s", gitHubEventTypePrefix, ghEventType)
+}
+
+// GitHubEventSource returns the GitHub CloudEvent source value.
+func GitHubEventSource(ownerAndRepo string) string {
```

Comment:
In the Registry discussions we agreed on using `github.com` as the CE source. 
IMO that is not useful, we need more info there. I'm adding the owner and the repo as well.
E.g., `//github.com/nachocano/eventing`.
Do we need the starting slashes? Added them to match GcpPubSub source

Reply:
I think consistency is good :)

Reply:
Should the source include the protocol, e.g. `https://github.com/...`? I don't know what the most recent thinking is on that.

Reply:
not sure. The CloudEvent spec example does have it. @n3wscott you are the CE guru? Shall we use `https://github.com/...` ? instead of `//github.com/...`

Reply:
made the change to include https:// as that is an example given is in the CloudEvents spec. 
If people don't agree, we can roll it back.